TeamRadarr.Radarr version 3.2.2.5080

alteredpark opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17762)

Commit:
TeamRadarr.Radarr version 3.2.2.5080

OfficialEsco commented on manifests/t/TeamRadarr/Radarr/3.2.2.5080/TeamRadarr.Radarr.installer.yaml:
```diff
@@ -0,0 +1,13 @@
+PackageIdentifier: TeamRadarr.Radarr
+PackageVersion: 3.2.2.5080
+Installers:
+- Architecture: x64
```
```suggestion
- Architecture: x86
```

OfficialEsco commented on manifests/t/TeamRadarr/Radarr/3.2.2.5080/TeamRadarr.Radarr.installer.yaml:
```diff
@@ -0,0 +1,13 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: TeamRadarr.Radarr
+PackageVersion: 3.2.2.5080
```
```suggestion
PackageVersion: 3.2.2
```
This also shows as 3.2.2 🤦‍♂️
![image](https://user-images.githubusercontent.com/15158490/122393086-cc8ed200-cf74-11eb-870f-42973f56b338.png)

OfficialEsco commented on manifests/t/TeamRadarr/Radarr/3.2.2.5080/TeamRadarr.Radarr.locale.en-US.yaml:
```diff
@@ -0,0 +1,13 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TeamRadarr.Radarr
+PackageVersion: 3.2.2.5080
```
```suggestion
PackageVersion: 3.2.2
```
![image](https://user-images.githubusercontent.com/15158490/122393121-d7496700-cf74-11eb-8083-7022f2526b58.png)

OfficialEsco commented on manifests/t/TeamRadarr/Radarr/3.2.2.5080/TeamRadarr.Radarr.locale.en-US.yaml:
```diff
@@ -0,0 +1,13 @@
+PackageIdentifier: TeamRadarr.Radarr
+PackageVersion: 3.2.2.5080
+PackageLocale: en-US
+Publisher: Team.Radarr
```
```suggestion
Publisher: Team Radarr
```

OfficialEsco commented on manifests/t/TeamRadarr/Radarr/3.2.2.5080/TeamRadarr.Radarr.yaml:
```diff
@@ -0,0 +1,9 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: TeamRadarr.Radarr
+PackageVersion: 3.2.2.5080
```
```suggestion
PackageVersion: 3.2.2
```

mikeparkie commented:
@OfficialEsco yep, understood now, I know that for next time. I'll re-raise if the devs update the packages.

OfficialEsco commented:
> @OfficialEsco yep, understood now, I know that for next time. I'll re-raise if the devs update the packages.

Does it matter that much tho? How often does it update under a version like 3.2.2?

mikeparkie commented:
Actually, that's a good point and thinking about it they all have auto updaters, so the short version isn't to bad. I'll accept those suggestions now 👍

Commit:
Update manifests/t/TeamRadarr/Radarr/3.2.2.5080/TeamRadarr.Radarr.yaml

Co-authored-by: Esco <OfficialEsco@users.noreply.github.com>

Commit:
Update manifests/t/TeamRadarr/Radarr/3.2.2.5080/TeamRadarr.Radarr.locale.en-US.yaml

Co-authored-by: Esco <OfficialEsco@users.noreply.github.com>

Commit:
Update manifests/t/TeamRadarr/Radarr/3.2.2.5080/TeamRadarr.Radarr.locale.en-US.yaml

Co-authored-by: Esco <OfficialEsco@users.noreply.github.com>

OfficialEsco commented:
I would recommend you to do these changes in GitHub Desktop as you need to change the folder name

Commit:
Compliance changes

Added x86/x64 into the manifest
Used short version (major) to match Windows installed version.

Commit:
Renamed folder versions

OfficialEsco commented on manifests/t/TeamRadarr/Radarr/3.2.2/TeamRadarr.Radarr.locale.en-US.yaml:
```diff
@@ -0,0 +1,12 @@
+PackageVersion: 3.2.2
+PackageLocale: en-US
+PackageUrl: https://radarr.video/
+Publisher: TeamRadarr
```
```suggestion
Publisher: Team Radarr
```

OfficialEsco reviewed (changes requested):
Ohhnoo so close 🤦‍♂️

Commit:
Update manifests/t/TeamRadarr/Radarr/3.2.2/TeamRadarr.Radarr.locale.en-US.yaml

Co-authored-by: Esco <OfficialEsco@users.noreply.github.com>

OfficialEsco reviewed (approved):
🎉
